### PR TITLE
feat!:  add playwright support

### LINKED ISSUE #139: Add playwright support
User should be able to choose if he wants to use playwright or puppeteer.
All exposed api should remain the same, if user wants underlying instance, it will get puppeteer or playwright with their API

### PR DESCRIPTION
**Pull request by mpetrunic**
**Short description of work done**

### PR Checklist

- [x] I have run linter locally
- [x] I have run unit and integration tests locally
  - [ ] Update configuration the newest version (readme and const)
- [x] Rebased to master branch / merged master

### Changes
Added playwright support.

BREAKING CHANGE:
- changed launch params
- abstracted browser, page and element
- playwright takes precedence

### Issues

Closes #139

### CODE REVIEW

File: src/helpers/selectors.ts [hunk 1]
```diff
@@ -45,22 +46,30 @@ export const getSettingsSwitch = (
...
 
   // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-  const errorElement: ElementHandle<HTMLElement> | null = await Promise.race([
-    page.waitForSelector(`span.error`, options),
-    page.waitForSelector(`.typography--color-error-1`, options),
-    page.waitForSelector(`.typography--color-error-default`, options),
-  ]).catch(() => null);
+  const errorElement: DappeteerElementHandle<Serializable> | null =
+    await Promise.race([
+      page.waitForSelector(`span.error`, options),
```
Comment (Lykhoyda) [inline src/helpers/selectors.ts:59]: As a suggestion, we can use testing-library to have similar selectors regardless of the framework or platform we are using. I find it annoying to learn all the different ways of searching for the element as implementations are different. So, why not use the library which is widely used across developers to have a similar API for querying the UI elements? 

https://testing-library.com/docs/pptr-testing-library/intro
https://github.com/testing-library/playwright-testing-library

File: src/helpers/actions.ts [hunk 2]
```diff
@@ -85,25 +91,24 @@ export const clickOnLogo = async (page: Page): Promise<void> => {
...
   }
 
-  if (clear)
-    await page.evaluate((node: HTMLInputElement) => {
-      node.value = "";
-    }, input);
+  if (clear) {
+    await input.type("");
+  }
   await input.type(text);
```
Comment (BeroBurny) [inline src/helpers/actions.ts:112]: not sure if type `await input.type("");` will clear the input field

File: src/helpers/selectors.ts (hunk 1 above)
Comment (mpetrunic) [reply src/helpers/selectors.ts:59]: Seems good, could you open issue with your suggested approach?

File: src/jest/teardown.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-export default async function (): Promise<void> {
-  // close the browser instance
-  await global.browser.close();
-}
+// export default async function (): Promise<void> {
+//   // close the browser instance
+//   await global.browser.close();
+// }
```
Comment (BeroBurny) [inline src/jest/teardown.ts:4]: if there is no usage it should be removed

File: src/helpers/actions.ts (hunk 2 above)
Comment (mpetrunic) [reply src/helpers/actions.ts:112]: modified respective implementations to delete on empty value

**Review by BeroBurny — APPROVED** [review]
:package:

**Pull request merged**